network/vpc/reference-content/understanding-routing.mdx

```diff
@@ -0,0 +1,90 @@
+
+<Lightbox src="scaleway-routing.webp" alt="A diagram shows two Private Networks inside a VPC. A route table on a virtual router sits inside the VPC, and the route table is synchronized to each resource attached to the VPC's Private Networks. An Instance on Private Network A is able to send a packet to an Instance on Private Network B. An Elastic Metal server on Private Network A is able to send a packet to the public internet via the Public Gateway also on Private Network A." />
+
+TODO DECIDE: VERSION 2: NO CONCEPT OF VROUTER
+
+The diagram below shows how routing works across two Private Networks on a VPC. The VPC's route table is synched to each resource as it joins a Private Network. An Elastic Metal server on Private Network A is therefore able to find the route to send a packet to the public internet via a Public Gateway also attached to Private Network A. An Instance also on Private Network A is able to send a packet to an Instance on Private Network B.
+
+<Lightbox src="scaleway-routing-alt.webp" alt="A diagram shows two Private Networks inside a VPC. A route table sits inside the VPC, and is synchronized to each resource attached to the VPC's Private Networks. An Instance on Private Network A is able to send a packet to an Instance on Private Network B. An Elastic Metal server on Private Network A is able to send a packet to the public internet via the Public Gateway also on Private Network A." />
+
+## Route table
+
+<Lightbox src="scaleway-route-table.webp" alt="A screenshot of a route table from the Scaleway console is shown, exemplifying the principles described in the text below." />
+
+Every VPC has an associated **route table**, used to manage and control the routing of traffic within the VPC. The routes within a route table tell the VPC where to send traffic trying to get to a specific destination IP address. One line in the route table corresponds to one route. A route consists of:
+
+- A **destination** IP or IP range. This specifies that the route applies to traffic with a matching destination IP.
+- A **next hop**. This specifies where the VPC should forward traffic that is trying to reach the destination IP. If the destination IP falls within the [CIDR block](#cidr-block) of a Private Network, its next hop will be that Private Network, where it will find the attached resource with the specific private IP address specified by the destination. If the destination IP is not known on the VPC (represented by the `0.0.0.0/0` address), its next hop will be a Public Gateway so that it can reach the public internet (as long as a Public Gateway set to advertise the default route has been attached to the Private Network).
```

Comment:
```suggestion
- A **next hop**. This specifies where the VPC should forward traffic that is trying to reach the destination IP. If the destination IP falls within the [CIDR block](#cidr-block) of a Private Network, its next hop will be that Private Network, where it will find the attached resource with the specific private IP address specified by the destination. If the destination IP is not known on the VPC (represented by the `0.0.0.0/0` address), its next hop will be a Public Gateway so that it can reach the public internet (as long as a Public Gateway set to advertize the default route has been attached to the Private Network).
```